src/Compiler/perf/Microbenchmarks/MSN.cshtml

```diff
@@ -1354,14 +1354,14 @@
     <li class=" single   hasimage  media photo"  data-aop="slide19>single "  data-id="161" data-m='{"i":161,"p":106,"n":"single","y":13,"o":19}' >
             <a 
          class="media photo"
-        href="/en-us/entertainment/music/the-latin-grammy-awards-15-years-of-style/ss-BBmZSa7"
+        href="/en-us/entertainment/music/the-music-awards-15-years-of-style/ss-BBmZSa7"
          data-id="162" data-m='{"i":162,"p":161,"n":"HeadlineItemViewModel","y":13,"l":"BBmZSa7","h":0,"v":"entertainment","c":"music","o":1}'  >
         
         
 
-<img alt="Sofia Vergara arrives at the 12th Annual Latin Grammy Awards on Thursday Nov. 10..." data-src="{&quot;default&quot;:&quot;//img-s-msn-com.akamaized.net/tenant/amp/entityid/BBmWirX.img?h=368&amp;w=622&amp;m=6&amp;q=60&amp;u=t&amp;o=t&amp;l=f&amp;f=jpg&amp;x=752&amp;y=348&quot;}" src="//static-hp-wus.s-msn.com/sc/9b/e151e5.gif" title="AP" />
+<img alt="The presenter arrives at the 112th Annual Music Awards on Thursday Nov. 10..." data-src="{&quot;default&quot;:&quot;//img-s-msn-com.akamaized.net/tenant/amp/entityid/BBmWirX.img?h=368&amp;w=622&amp;m=6&amp;q=60&amp;u=t&amp;o=t&amp;l=f&amp;f=jpg&amp;x=752&amp;y=348&quot;}" src="//static-hp-wus.s-msn.com/sc/9b/e151e5.gif" title="AP" />
```

Comment:
was it required to change this?

Reply:
Looks like avoiding potential policheck issues?